doc/moveit_cpp/config/panda_controllers.yaml

```diff
@@ -0,0 +1,18 @@
+panda_arm_controller:
```

Comment:
This's for the old ros2_control, we're depending on the master branch now, see [run_moveit_cpp.launch.py](https://github.com/ros-planning/moveit2/blob/main/moveit_demo_nodes/run_moveit_cpp/launch/run_moveit_cpp.launch.py)